crates/ruff/resources/test/fixtures/flake8_i18n/INT001.py

```diff
@@ -0,0 +1 @@
+_(f"{'value'}")
```

Comment:
`f"{'value'}"` is just the string "value".  I'm not sure if the AST will pick up that.  In general, any f-string inside a i18n function might be problematic. 

```suggestion
_(f"{value}")
```

Reply:
Yes, it will, any f-string is a JoinedString for the parser.